dashboard/app/jobs/aichat_safety_job.rb

```diff
@@ -0,0 +1,17 @@
+require "cdo/job_with_results"
+
+class AichatSafetyJob < JobWithResults
+  self.queue_adapter = :delayed_job
+  queue_as :default
+
+  @@timeout_s = 15
+
+  def perform(prompt:)
+    results = Python.run do
+      pyimport "pycdo.aichat.safety", as: :safety
```

Comment:
Currently this will CRASH HERE. LLMGuard plays pretty rough. This PR is blocking on me fixing this underlying code.